packages/files-ui/src/Utils/pathUtils.ts

```diff
@@ -22,6 +22,14 @@ export function getArrayOfPaths(path: string): string[] {
   }
 }
 
+// "/" -> []
+// "/path/to/this" -> ["path", "to", "this"]
```

Comment:
If I'm not mistaking, should update the example to something like
```suggestion
// [] -> "/"
// ["path", "to", "this"] => "/path/to/this"
```